enginetest/memory_engine_test.go

```diff
@@ -200,43 +200,6 @@ func TestSingleScript(t *testing.T) {
 
 		enginetest.TestScriptWithEngine(t, engine, harness, test)
 	}
-	//t.Skip()
```

Comment:
Thanks for tidying this up!! I noticed this messiness the other day in my workspace, too.